C06.csv

```diff
@@ -139,6 +139,27 @@ CodeFigure,UnitType,Meaning,conventional,IA5-ASCII,ITA2,SIDefinition,Note,NoteID
 842,Other Units as used by WMO,N units,N units,N units,,,,,Operational
 843,Other Units as used by WMO,Nephelometric turbidity units,NTU,,,,,,Operational
 844,Other Units as used by WMO,Total electron content unit,TECU,,,10^16 Electrons m^-2,,,
+845,Other Units as used by WMO,per Kelvin,K^-1,K-1,,,,,
+846,Other Units as used by WMO,square Kelvin,K^2,K2,,,,,
+847,Other Units as used by WMO,logarithm square metre per square metre,log(m^2 m^-2),log(m2 m-2),,,,,
+848,Other Units as used by WMO,logarithm cubic metre per square metre,log(m^3 m^-2),log(m3 m-2),,,,,
+849,Other Units as used by WMO,kilograms per square metre per hour,kg m^-2 h^-1,kg m-2 h-1,,,,,
+850,Other Units as used by WMO,Kelvin per Kelvin,K K^-1,K K-1,,,,,
+851,Other Units as used by WMO,Watts per square metre per steradian per micro-metre,W m^-2 sr^-1 um^-1,W m-2 sr-1 um-1,,,,,
+852,Other Units as used by WMO,Candela per square metre,Cd m^-2,Cd m-2,,,,,
+853,Other Units as used by WMO,logarithm per cubic metre,log(m^-3),log(m-3),,,,,
+854,Other Units as used by WMO,per square metre,m^-2,m-2,,,,,
+855,Other Units as used by WMO,per metre per steradian,m^-2 sr^-1,m-2 sr-1,,,,,
```

Comment:
There's an error in the original proposal, it should be `m-1` not `m-2`, i.e.

`855,Other Units as used by WMO,per metre per steradian,m^-1 sr^-1,m-1 sr-1,,,,,`